src/Nest/QueryDsl/TermLevel/Wildcard/WildcardQuery.cs

```diff
@@ -28,7 +28,7 @@ public class WildcardQuery : FieldNameQueryBase, IWildcardQuery
 		protected override bool Conditionless => TermQuery.IsConditionless(this);
 		public object Value { get; set; }
 
-		[Obsolete("Use MultiTermQueryRewrite")]
+		[Obsolete("Use MultiTermQueryRewrite, deleted in NEST 6.x")]
```

Comment:
I think `fixed` is a better description here; The property `Rewrite` exists in NEST 6.x, but it is of type `MultiTermQueryRewrite`.